server/src/domain/schema.rs

```diff
@@ -13,26 +13,6 @@ impl PublicSchema {
     }
 }
 
-pub trait SchemaAttributeExtractor: std::marker::Send {
```

Comment:
That's nice :) It felt a bit like an over-abstraction, to be fair

Reply:
Seemed it got a bit pointless when I had to add other parameters anyway. :-)